src/locales/de.ts

```diff
@@ -10,42 +10,42 @@ export default {
     'general.signOut': 'Abmelden',
     'general.save': 'Speichern',
     'general.default': 'Standard',
-    'general.networkError': 'Netzwerk-Fehler',
+    'general.networkError': 'Netzwerkfehler',
 
     // Storage
-    'storage.name': 'Speicherort Name',
+    'storage.name': 'Name des Speicherorts',
     'storage.noStorage': 'Kein Speicherort',
     'storage.create': 'Speicherort erstellen',
     'storage.edit': 'Speicherort ändern',
     'storage.rename': 'Speicherort umbenennen',
-    'storage.renameMessage': 'Name für Speicherort ändern',
+    'storage.renameMessage': 'Name des Speicherorts ändern',
     'storage.remove': 'Speicherort entfernen',
-    'storage.removeMessage': 'Speicherort wird von App entknüpft',
+    'storage.removeMessage': 'Speicherort wird gelöscht',
```

Comment:
are they really deleted? In the old app they where merely unlinked. `gelöscht` would cause confusion, then. 

If they aren't, which I don't know if that is the case, then `Verknüpfung zum Speicherort entfernen` would be better.